pkg/tracing/tracing_test.go

```diff
@@ -113,6 +114,17 @@ func TestPropagateSpan(t *testing.T) {
 	}
 }
 
+func TestPropagateSpanError(t *testing.T) {
+	tracer := NewTracer(mocktracer.New())
+	tracer.inject = func(sm ot.SpanContext, format interface{}, carrier interface{}) error {
+		return errors.New("error")
```

Comment:
Oh whoops, yup you're correct. Feels odd that nowhere in this test do we have a failure condition - how do we know short of a panic that things are working as intended?